cucumber/policy/features/deletion.feature

```diff
@@ -264,3 +270,33 @@ Feature: Deleting objects and relationships.
     Then the role list includes host "host-01"
     And the role list includes host "host-02"
     And the role list includes host "host-03"
+
+    @smoke
+    Scenario: Privileges removed on opposite policy statements
+      Given I update the policy with:
+      """
+      - !variable db/password
+      - !host host-01
```

Comment:
It would be good to flesh this out a little to make it clear what we're verifying with this test.

```suggestion
    Scenario: Privileges removed on conflicting policy statements
      
      If a policy contains statements to both add and remove a privilege
      (permit/deny, grant/revoke), then we want to ensure that we fail
      safe and the the negative privilege statement is the final outcome.
    
      Given I update the policy with:
      """
      - !variable db/password
      - !host host-01
```